templates/vault_backend_raft.j2

```diff
@@ -0,0 +1,4 @@
+storage "raft" {
+  path = "{{ vault_data_path }}"
+  node_id = "{{ vault_raft_node_id }}"
+}
```

Comment:
Missing newline

Reply:
raft_node_id can't be the same in a multinode cluster.

Reply:
The spaces are fixed now. About **raft_node_id** this need to be diferent on every node/instace that can be specified via hosts_vars.